init.d.optee

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# /etc/init.d/optee
+#
+# Start/stop tee-supplicant (OP-TEE normal world daemon)
+#
+case "$1" in
+    start)
+	if [ -e /bin/tee-supplicant -a -e /dev/teepriv0 ]; then
```

Comment:
Indeed, I'd rather keep this simple. Here we are targeting a controlled environment (our busybox-based rootfs) and we know that the TEE should be listening on `/dev/teepriv0`.